platform/darwin/src/NSExpression+MGLAdditions.mm

```diff
@@ -62,6 +62,7 @@ + (void)installFunctions {
     INSTALL_METHOD(mgl_step:from:stops:);
     INSTALL_METHOD(mgl_coalesce:);
     INSTALL_METHOD(mgl_does:have:);
+    INSTALL_METHOD(mgl_expression:betweenLeftHandExpression:rightHandExpression:);
```

Comment:
It’s unnecessary to provide a custom expression function for the `BETWEEN` operator. This operator is specific to NSPredicate and can’t be used inside an NSExpression. There’s already a way to use the `BETWEEN` operator without a format string:

```objc
[NSComparisonPredicate predicateWithLeftExpression:… rightExpression:… modifier:NSDirectPredicateModifier type:NSBetweenPredicateOperatorType options:0]
```

In general, we should be using NSPredicate itself, rather than NSExpression, to represent Boolean-typed mbgl expressions, which is why I didn’t bother to create an aftermarket function for converting to Boolean values in #11472.